java-client/src/main/java/org/opensearch/client/opensearch/core/MsearchRequest.java

```diff
@@ -321,6 +341,17 @@ public final Builder allowNoIndices(@Nullable Boolean value) {
             return this;
         }
 
+        /**
+         * The time after which the search request will be canceled.
+         * Request-level parameter takes precedence over cancel_after_time_interval cluster setting.
+         * <p>
+         * API name: {@code cancel_after_time_interval}
+         */
+        public final Builder cancelAfterTimeInterval(@Nullable Time value) {
```

Comment:
This should also have the builder `Function` overload like you've added to `SearchRequest`

Reply:
@Xtansia , added the function overload method.
I did not added initially becase no other method has function overloads.